nat-lab/tests/test_vpn.py

```diff
@@ -35,6 +35,22 @@ async def _connect_vpn(
     assert ip == wg_server["ipv4"], f"wrong public IP when connected to VPN {ip}"
 
 
+async def _connect_vpn_nordlynx(
```

Comment:
is there actually a need for this function?

Reply:
So basically it was duplicated because existing tests pinged the client from the server which I am not sure how to configure so it would work on nordlynx server. So I duplicated the function and removed one ping instead. Now as I parametrizes everything there's no need for this function anymore I suppose but as a simplification I also removed the offending ping from the code